Source/ACE/Entity/Player.cs

```diff
@@ -1,1226 +1,1231 @@
+            Sequences.AddOrSetSequence(SequenceType.PrivateUpdatePropertyInt64, new ByteSequence(false));
+            Sequences.AddOrSetSequence(SequenceType.PrivateUpdatePropertyDouble, new ByteSequence(false));
+            Sequences.AddOrSetSequence(SequenceType.PrivateUpdatePropertyString, new ByteSequence(false));
+
+            // This is the default send upon log in and the most common.   Anything with a velocity will need to add that flag.
+            PositionFlag |= UpdatePositionFlag.ZeroQx | UpdatePositionFlag.ZeroQy | UpdatePositionFlag.Contact | UpdatePositionFlag.Placement;
+
+            Name = session.CharacterRequested.Name;
+            Icon = 0x1036;
+            GameData.ItemCapacity = 102;
+            GameData.ContainerCapacity = 7;
+            GameData.RadarBehavior = RadarBehavior.ShowAlways;
+            GameData.RadarColour = RadarColor.White;
+            GameData.Usable = Usable.UsableObjectSelf;
+
+            SetPhysicsState(PhysicsState.IgnoreCollision | PhysicsState.Gravity | PhysicsState.Hidden | PhysicsState.EdgeSlide, false);
+            PhysicsData.PhysicsDescriptionFlag = PhysicsDescriptionFlag.CSetup | PhysicsDescriptionFlag.MTable | PhysicsDescriptionFlag.Stable | PhysicsDescriptionFlag.Petable | PhysicsDescriptionFlag.Position | PhysicsDescriptionFlag.Movement;
+
+            // apply defaults.  "Load" should be overwriting these with values specific to the character
+            // TODO: Load from database should be loading player data - including inventroy and positions
+            PhysicsData.CurrentMotionState = new GeneralMotion(MotionStance.Standing);
+            PhysicsData.MTableResourceId = 0x09000001u;
+            PhysicsData.Stable = 0x20000001u;
+            PhysicsData.Petable = 0x34000004u;
+            PhysicsData.CSetup = 0x02000001u;
+
+            // radius for object updates
+            ListeningRadius = 5f;
+        }
+
+        /// <summary>
+        ///  Gets a list of Tracked Objects.
+        /// </summary>
+        public List<ObjectGuid> GetTrackedObjectGuids()
+        {
+            lock (clientObjectMutex)
+            {
+                return clientObjectList.Select(x => x.Key).ToList();
+            }
+        }
+
+        public void Kill()
+        {
+            // create corpse at location
+            // TODO: Once the corpse/container factories have been built
+
+            // teleport to sanctuary or starting point
+            if (Positions.ContainsKey(PositionType.Sanctuary))
+                Teleport(Positions[PositionType.Sanctuary]);
+            else
+                Teleport(Positions[PositionType.Location]);
+
+            // create and send the death event
+            var yourDeathEvent = new GameEventYourDeath(Session);
+            Session.Network.EnqueueSend(yourDeathEvent);
+        }
+
+        public async Task Load(Character preloadedCharacter
```

Comment:
I killed this line in PR #268. it's doing more harm than good. A great example of that is to `@telepoi holtburg` and run to the portal down the hill, walk backwards from it and watch objects blink as they get repopulated for no good reason. it didn't work like that retail, it shouldn't here either.

I know you didn't add this.. just noting it for you